.vscode/settings.json

```diff
@@ -60,7 +60,7 @@
     },
     "particle.firmwareVersion": "5.3.0",
     "particle.targetPlatform": "tracker",
-    "particle.targetDevice": "",
+    "particle.targetDevice": "e00fce68432f6ede72157b61",
```

Comment:
This change is probably not something we should commit, since it ties the repo to a specific device